generated/nidcpower/nidcpower.proto

```diff
@@ -1478,6 +1679,44 @@ message GetExtCalRecommendedIntervalResponse {
   sint32 months = 2;
 }
 
+message GetLCRCompensationDataRequest {
+  nidevice_grpc.Session vi = 1;
+  string channel_name = 2;
+}
+
+message GetLCRCompensationDataResponse {
+  int32 status = 1;
+  bytes compensation_data = 2;
+}
+
+message GetLCRCompensationLastDateAndTimeRequest {
+  nidevice_grpc.Session vi = 1;
+  string channel_name = 2;
+  oneof compensation_type_enum {
+    LCRCompensationType compensation_type = 3;
+    sint32 compensation_type_raw = 4;
+  }
+}
+
+message GetLCRCompensationLastDateAndTimeResponse {
+  int32 status = 1;
+  sint32 year = 2;
```

Comment:
It's funny to have `sint32` for these because they can't be negative (*unless ....*) . But I assume that this is a static mapping so it's probably not worth fixing. I'm sure we goof the other way plenty of times, it's just less obvious.

Reply:
True, at least in this case, though, this matches the type in the header for `niDCPower_GetLCRCompensationLastDateAndTime`. The signature looks like

```c
ViStatus _VI_FUNC niDCPower_GetLCRCompensationLastDateAndTime(
      ViSession vi,
      ViConstString channelName,
      ViInt32 compensationType,
      ViInt32* year,
      ViInt32* month,
      ViInt32* day,
      ViInt32* hour,
      ViInt32* minute);
```

So who knows, maybe the last measurement was B.C.  or at least before 1970